explorer/src/lib/components/blocks-list/BlocksList.svelte

```diff
@@ -74,11 +74,11 @@
 
   <!-- BLOCK REWARD -->
   <ListItem tooltipText="The reward allocated to the block generator">
-    <svelte:fragment slot="term">rewards</svelte:fragment>
+    <svelte:fragment slot="term">rewards (dusk)</svelte:fragment>
```

Comment:
I noticed this isn't in the Figma design @laremas

Reply:
This has been removed after a discussion with @laremas